kms++/src/plane.cpp

```diff
@@ -109,12 +109,12 @@ uint32_t Plane::fb_id() const
 	return m_priv->drm_plane->fb_id;
 }
 
-uint32_t Plane::crtc_x() const
+int32_t Plane::crtc_x() const
 {
 	return m_priv->drm_plane->crtc_x;
```

Comment:
m_priv->drm_plane->crtc_x is unsigned, I wonder if this is correct... This is for the non-atomic modesetting, and kms++ doesn't currently even us these functions. Maybe it's better to leave this and the crtc_y below as they are now.

The kmstest part looks fine.